### ISSUE TITLE
발견한 embedding 결과를 다른 사람과 공유하는 것은 괜찮을까요?

### ISSUE THREAD

**Issue by ChanwooLucioLee** [SNUCSE-CTA/Graph-Pattern-Matching-Challenge#29]
안녕하세요,

과제 1에서 test case를 공유했던 것과 유사하게 코드 자체가 아닌 코드를 돌린 결과를 공유하는 것이 이번 과제에서도 허용되는지 궁금합니다.

구체적으로 말씀드리면 respository에 있는 data/query graph에 대해 발견한 (correct) embedding 결과에 대해 다른 사람들과 공유하는 것이 가능할까요?

감사합니다.

**Comment by JihoonJang** [comment]
네, 코드를 돌린 결과만 공유하는 것은 괜찮습니다. (다만 코드 구현이나 아이디어를 공유하는 것은 금지되어 있습니다.)

감사합니다.

**Comment by ChanwooLucioLee** [comment]
네 답변 감사드립니다!
